react-app/src/app/screens/LoginFormContainer/components/LoginForm/index.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+import PropTypes from 'prop-types';
+
+import { required, minLength, validateEmail } from '../../validation';
+
+import { customInput } from './fields';
+import './fields/LoginForm.css';
+
+const LoginForm = ({ handleSubmit }) => (
+  <form className="form-container" onSubmit={handleSubmit}>
+    <Field
+      name="email"
+      component={customInput}
+      type="text"
+      label="Email"
+      validate={[required, validateEmail]}
+    />
+    <Field
+      name="password"
+      component={customInput}
+      type="password"
+      label="Password"
+      validate={[required, minLength]}
+    />
+    <button className="button" type="submit">
+      Submit
+    </button>
+  </form>
+);
+
+LoginForm.propTypes = {
+  handleSubmit: PropTypes.func.isRequired
+};
+
+export default reduxForm({
+  form: 'login'
```

Comment:
`login` `Password` `Email` `Submit` Should be declare as constants in a constant file